File: 2239.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var findClosestNumber = function(nums) {
    let mem = 999999
    nums.sort()
    for (let i = 0; i < nums.length(); i++) {
        if (Math.abs(nums[i]) <= Math.abs(mem)) {
            mem = Math.abs(nums[i])
        }
    }
};


findClosestNumber([-4,-2,1,4,8])

File: 2239.py
Sample = [-4,-2,1,4,8]
class Solution:
    def question(self,nums:list) -> int: 
        nums.sort()
        mem = 9999
        for i in nums:
            print (abs(i))
            if abs(i) <= abs(mem):
                mem = i
        return mem



if __name__ == '__main__':
    hold = (Solution.question(None,Sample))
    print (hold)